plc4j/spi/src/main/java/org/apache/plc4x/java/spi/transport/Transport.java

```diff
@@ -28,4 +28,8 @@ public interface Transport {
 
     ChannelFactory createChannelFactory(String transportConfig);
 
+    default Class<? extends TransportConfiguration> getTransportConfigType() {
+        return null;
```

Comment:
we could also consider the use of optional, depending on how it is use (not a issue with null)